packages/pharos/package.json

```diff
@@ -84,7 +84,20 @@
     "sinon": "^12.0.1",
     "style-dictionary": "^3.0.1",
     "ts-lit-plugin": "^1.2.1",
-    "typescript": "^4.5.2"
+    "typescript": "^4.9.4"
   },
-  "customElements": "custom-elements.json"
+  "customElements": "custom-elements.json",
+  "exports": {
+    ".": "./lib/index.js",
+    "./lib": "./lib/index.js",
+    "./lib/index": "./lib/index.js",
+    "./lib/index.js": "./lib/index.js",
+    "./lib/react-components": "./lib/react-components/index.js",
+    "./lib/styles/pharos": "./lib/styles/pharos.scss",
+    "./lib/styles/pharos.scss": "./lib/styles/pharos.scss",
+    "./lib/utils/scss/mixins": "./lib/utils/scss/_mixins.scss",
```

Comment:
`yarn build:site` is failing because when it tries to build, it expects other files to be exported. Using patterns would be useful here if really do need to import all of these files https://nodejs.org/api/packages.html#subpath-patterns

Running site build locally passed with warnings instead of errors:

```diff
--- a/packages/pharos/package.json
+++ b/packages/pharos/package.json
@@ -95,7 +95,22 @@
     "./lib/react-components": "./lib/react-components/index.js",
     "./lib/styles/pharos": "./lib/styles/pharos.scss",
     "./lib/styles/pharos.scss": "./lib/styles/pharos.scss",
+    "./lib/styles/typography": "./lib/styles/pharos.scss",
+    "./lib/styles/typography.scss": "./lib/styles/typography.scss",
+    "./lib/styles/fonts.css": "./lib/styles/fonts.css",
+    "./lib/styles/variables": "./lib/styles/variables.css",
+    "./lib/styles/variables.css": "./lib/styles/variables.css",
+    "./lib/styles/variables.js": "./lib/styles/variables.js",
+    "./lib/styles/pharos-alert.css": "./lib/styles/pharos-alert.css",
+    "./lib/styles/pharos-footer.css": "./lib/styles/pharos-footer.css",
+    "./lib/styles/pharos-modal.css": "./lib/styles/pharos-modal.css",
+    "./lib/styles/tokens": "./lib/styles/tokens.js",
+    "./lib/styles/tokens.js": "./lib/styles/tokens.js",
     "./lib/utils/scss/mixins": "./lib/utils/scss/_mixins.scss",
+    "./lib/utils/registerComponents": "./lib/utils/registerComponents.js",
+    "./lib/utils/registerComponents.js": "./lib/utils/registerComponents.js",
+    "./lib/utils/PharosContext": "./lib/utils/PharosContext.js",
+    "./lib/utils/PharosContext.js": "./lib/utils/PharosContext.js",
     "./lib/utils/scss/mixins.scss": "./lib/utils/scss/_mixins.scss",
     "./package.json": "./package.json",
     "./custom-elements.json": "./custom-elements.json"
```